packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -284,6 +291,51 @@ export function createTestEsCluster<
       );
 
       log.info('[es] stopped');
+
+      await this.captureDebugFiles();
+    }
+
+    async captureDebugFiles() {
+      const debugFiles = await globby([`**/hs_err_pid*.log`, `**/replay_pid*.log`, `**/*.hprof`], {
+        cwd: config.installPath,
+        absolute: true,
+      });
+
+      if (!debugFiles.length) {
+        log.info('[es] no debug files found, assuming es did not write any');
+        return;
+      }
+
+      const uuid = Uuid.v4();
+      const debugPath = Path.resolve(KIBANA_ROOT, `data/es_debug_${uuid}.tar.gz`);
```

Comment:
Thoguhts on adding this to package.json kibana.clean.extraPatterns?

Reply:
Sounds reasonable

Reply:
Ah nevermind, I see this isn't triggered locally.